cpp/memilio/mobility/graph.h

```diff
@@ -178,6 +178,62 @@ class Graph
                                       });
     }
 
+    /**
+    * @brief Add edges to a graph without checking for duplicates and without sorting.
+    * 
+    * @param start_node_idx Id of start node
+    * @param end_node_idx Id of end node
+    * @param args Additional arguments for edge construction
+    * @return Edge<EdgePropertyT>& End of edge vector
+    */
+    template <class... Args>
+    Edge<EdgePropertyT>& lazy_add_edge(size_t start_node_idx, size_t end_node_idx, Args&&... args)
```

Comment:
Can you also add a motivation, why we even need this function? Whats the use case? why not add a bool flag to the existing add_edge function to allow addition without checking or sorting?

also, i would prefer the name `add_edge_naive`

Reply:
or maybe directly `unchecked`?